config/cesm/machines/config_machines.xml.stampede

```diff
@@ -0,0 +1,2119 @@
+<?xml version="1.0"?>
```

Comment:
Why is this file here?

Reply:
I accidentally added it.  It's removed now.